itest/litd_mode_integrated_test.go

```diff
@@ -198,16 +201,16 @@ var (
 		restWebURI:                  "/v1/pool/info",
 	}, {
 		name:       "litrpc",
-		macaroonFn: nil,
+		macaroonFn: litMacaroonFn,
 		requestFn:  litRequestFn,
 		// In some test cases we actually expect some sessions, so we
 		// don't explicitly check for an empty array but just the
 		// existence of the array in the response.
 		successPattern:              "\"sessions\":[",
-		supportsMacAuthOnLndPort:    false,
-		supportsMacAuthOnLitPort:    false,
-		supportsUIPasswordOnLndPort: true,
-		supportsUIPasswordOnLitPort: true,
+		supportsMacAuthOnLndPort:    true,
```

Comment:
All these boolean variables can now be removed since they should always be `true, true, false, true` respectively with litrpc having caught up with the rest of the sub RPC servers.